apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { MultiGrid } from 'react-virtualized';
+import ProgressBubble from '../progress/ProgressBubble';
+
+const STYLE = {
+  border: '1px solid #ddd',
+};
+const STYLE_BOTTOM_LEFT_GRID = {
+  borderRight: '2px solid #aaa',
+  backgroundColor: '#f7f7f7',
+};
+const STYLE_TOP_LEFT_GRID = {
+  borderBottom: '2px solid #aaa',
+  borderRight: '2px solid #aaa',
+  fontWeight: 'bold',
+};
+const STYLE_TOP_RIGHT_GRID = {
+  borderBottom: '2px solid #aaa',
+  fontWeight: 'bold',
+};
+
+export default class VirtualizedDetailView extends Component {
+
+  constructor(props, context) {
+    super(props, context);
```

Comment:
I'm not sure I understand the reason for an explicit constructor and call to super here. What's it for?